### PR TITLE
Usability improvements

### LINKED ISSUE #149: Add documentation to model keys
Add documentation similar to that on the people and features model.

`Help_text` attribute to model fields.

### PR DESCRIPTION
**Pull request by alanmoo**
Closes #149, #139, partially addresses #164 (enough to launch)

### CODE REVIEW

File: app/networkapi/landingpage/models.py [hunk 1]
```diff
@@ -4,38 +4,53 @@
...
 
     # featured-image = s3boto3 stuffs
 
-    featured = models.BooleanField(default=False)
+    # featured = models.BooleanField(default=False)
+
+    header = models.CharField(
+        max_length=500,
+        help_text="Page title, appears above content",
+        )
```
Comment (Pomax) [inline app/networkapi/landingpage/models.py]: too much indent

File: app/networkapi/settings.py [hunk 2]
```diff
@@ -203,6 +202,15 @@
     'mezzanine.generic.models.ThreadedComment',
 ]
 
+ADMIN_MENU_ORDER = (
+    ('Content', ('pages.Page', 'blog.BlogPost', 'generic.ThreadedComment',
```
Comment (Pomax) [inline app/networkapi/settings.py]: can we remove blog and comment, as we don't use them.

File: app/networkapi/settings.py [hunk 3]
```diff
@@ -203,6 +202,15 @@
...
+                 ('Media Library', 'media-library'))),
+    ('Site', ('sites.Site', 'redirects.Redirect', 'conf.Setting')),
```
Comment (Pomax) [inline app/networkapi/settings.py]: remove conf.settings? Either that, or we need to make sure that only users with admin rights can see that section (as a followup)

File: app/networkapi/settings.py [hunk 4]
```diff
@@ -203,6 +202,15 @@
...
+    ('Users', ('auth.User', 'auth.Group')),
+    ('Data', ('people.Person', 'news.News', 'people.InternetHealthIssue',)),
+    ('Components', ('landingpage.Signup',)),
```
Comment (Pomax) [inline app/networkapi/settings.py]: Something we should consider is renaming signup to "CTA block" or something

File: app/networkapi/urls.py [hunk 5]
```diff
@@ -25,7 +25,6 @@
 urlpatterns = [
     url('^admin/', include(admin.site.urls)),
     url(r'^api/people/', include('networkapi.people.urls')),
-    url(r'^api/features/', include('networkapi.features.urls')),
```
Comment (Pomax) [inline app/networkapi/urls.py]: inconvenient as it sounds, let's not overload PRs: remove features in a different PR

File: app/networkapi/settings.py [hunk 6]
```diff
@@ -83,7 +83,6 @@
     'adminsortable',
 
     'networkapi.people',
-    'networkapi.features',
```
Comment (Pomax) [inline app/networkapi/settings.py]: inconvenient as it sounds, let's not overload PRs: remove features in a different PR

File: app/networkapi/landingpage/models.py [hunk 7]
```diff
@@ -4,38 +4,53 @@
...
 
-    header = models.CharField(max_length=500)
     content = RichTextField("Main body content")
 
     signup = models.ForeignKey(
         Signup,
         related_name='page',
         null=True,
         on_delete=models.SET_NULL,
+        help_text="Choose an existing or create a new mailing list signup form"
```
Comment (Pomax) [inline app/networkapi/landingpage/models.py]: missing trailing comma

File: app/networkapi/landingpage/models.py (hunk 1 above)
Comment (alanmoo) [inline app/networkapi/landingpage/models.py]: Based on what? My linter complains if I have less

File: app/networkapi/settings.py (hunk 3 above)
Comment (alanmoo) [inline app/networkapi/settings.py]: It already behaves that way- I created a new user as a test and made them staff, but they couldn't see the settings. Lines 206-208 are simply the default value for this property.

Comment (cadecairos) [inline app/networkapi/settings.py]: lets leave it in. super users and admins will only be able to see it.

**Pull request merged**